src/UserSettings.hs

```diff
@@ -42,3 +42,7 @@ putBuild = putColoured Dull Magenta
 -- | Customise build success messages (e.g. a package is built successfully).
 putSuccess :: String -> Action ()
 putSuccess = putColoured Dull Green
+
+-- | Path to the GHC install destination
+defaultDestDir :: FilePath
+defaultDestDir = ""
```

Comment:
What is the semantics of the empty directory? Where do we install by default? Is there a sensible default? If not, should it be `Maybe FilePath`?